src/App.vue

```diff
@@ -183,7 +183,7 @@ export default defineComponent({
       userDropdownActive,
       createDropdownActive,
       ownerSettingsLink,
-
+      ownerProjectsLink,
```

Comment:
Is this needed?

Reply:
> Is this needed?

Do you mean to remove ownerProjectsLink?
I think it's needed the users's project page.

Reply:
This patch adds it. But looks like it isn't used by this patch.

Reply:
> This patch adds it. But looks like it isn't used by this patch.

Correct! I'll fix it.
Thanks